subjects/ai/visualizations/audit/README.md

```diff
@@ -1,40 +1,38 @@
 #### Exercise 0: Environment and libraries
 
-##### The exercice is validated is all questions of the exercice are validated
+##### The exercise is validated if all questions of the exercise are validated
 
 ##### Activate the virtual environment. If you used `conda` run `conda activate your_env`
 
 ##### Run `python --version`
 
 ###### Does it print `Python 3.x`? x >= 8
 
-##### Does `import jupyter`, `import numpy`, `import pandas`, `matplotlib` and `plotly` run without any error ?
+##### Do `import jupyter`, `import numpy`, `import pandas`, `matplotlib` and `plotly` run without any error ?
```

Comment:
should be an `h6`, also there is an extra space before `?`

Reply:
Thanks, corrected :grin: